manage/upgrading/version_specific_migration/upgrade_to_python3.rst

```diff
@@ -15,27 +15,26 @@ Principles
 ----------
 
 * You should support Python 2 and 3 with the same codebase to allow it to be used in existing versions of Plone.
-* Plone 5.2 supports Python 2.7, Python 3.6 and Python 3.7
-* We use `six <https://six.readthedocs.io>`_ and
-  `modernize <https://pypi.python.org/pypi/modernize>`_ to do the first steps towards Python 3.
+* Plone 5.2 supports Python 2.7, Python 3.6, and Python 3.7.
+* We use `six <https://six.readthedocs.io>`_ and `modernize <https://pypi.python.org/pypi/modernize>`_ to do the first steps towards Python 3.
 
 In general you should follow these steps to port add-ons:
 
-#. Prepare buildout for the add-on to be ported
-#. Update code with python-modernize
-#. Use precompile to find syntax errors
-#. Start the instance and find more errors
-#. Test functionality manually
-#. Run and fix all tests
-#. Update package information
+#. Prepare ``buildout`` for the add-on to be ported.
+#. Update code with `python-modernize <https://python-modernize.readthedocs.io/en/latest/>`_.
+#. Use `plone.recipe.precompiler <https://github.com/plone/plone.recipe.precompiler>`_ (also called ``precompiler`` for brevity) to find syntax errors.
+#. Start the instance and find more errors.
+#. Test functionality manually.
+#. Run and fix all tests.
+#. Update package information.
 
 
 1 Preparation
 -------------
 
 In the GitHub repository of the add-on:
 
-* Open a ticket with the title "Add support for Python 3" .
+* Open a ticket with the title "Add support for Python 3".
 * Create a new branch named ``python3``.
 
 Until Plone 5.2 is released, you can use the coredev buildout setup.
```

Comment:
This are supposed to be the docs for Plone 5.2, meaning this line 40 should be removed.

Reply:
If it is removed, then the subsequent sentence will not make sense. Perhaps just remove the phrase "Until Plone 5.2 is released, " and capitalize "you"?

Reply:
Maybe I am not understanding it?
If I look at the PR request, this is a request to merge it into the 5.2 branch.

Besides that, the branch is public on GitHub there is no other release of this branch.
AFAIK we will release it to docs.plone.org during the release of Plone 5.2.

Because of this, I am wondering why we need this line in the docs.
For me, this would make only sense if these docs would be merged into current 'master' (HTML) on docs.plone.org.

Do I not getting something?

Reply:
I agree about v5.2.

However, lines 40-53 all refer to using "coredev buildout setup". Removing all of line 40 will make lines 41-53 difficult to understand.

Reply:
OK, just read the whole part in a different browser tab.
Yes, that is indeed a problem!
The question for me is now if that part should be there like this at all?
The current part makes complete sense if you work on this before 5.2 is released, but after the release, this is (as I understand it0 obsolete (lite it is written now) in the 5.2 docs.
So we need a solution where we have now the current part, but after the release, that needs to change.
What is with the 'dirty' idea to keep it like it is for now and open a ticket (with a reminder) to change that after the release?
I can put it on my release checklist for example.

Reply:
We could put a `..warning::` preceding the section:

```rst
..warning::

    The following section is valid until the final release of Plone 5.2.
    Upon the final release of Plone 5.2, [something else will take its place].
```

We could also use `..todo::` or comment `..` in reST.

Reply:
Yes! I would go with the warning, awesome idea!

Reply:
Fixed in f1a5785. Anything else need to be noted, new issue, added to checklist?

Reply:
Added to checklist! Will polish the checklist a bit and will add it to GitHub sometime during next week.
Guess it is handy that more people than two, know the steps and how to fix common issues during our currently somewhat hacky and old way.
Well from Plone 6 on that will be way better.